.travis.yml

```diff
@@ -80,6 +80,18 @@ jobs:
     - cd rasa-website
     - git commit --allow-empty -m "trigger nlu docs update"
     - git push origin master
+  - stage: Test NLU starter pack
+    python: 3.6
+    script:
+    - git clone https://github.com/RasaHQ/starter-pack-rasa-nlu.git
+    - cd starter-pack-rasa-nlu
+    - py.test test_nlu.py
```

Comment:
can we put the `test_nlu.py` into a folder, e.g. `tests` and run `py.test tests`?